experiment/builder_runner.py

```diff
@@ -376,12 +378,19 @@ def build_and_run(self, generated_project: str, target_path: str,
                cwd=oss_fuzz_checkout.OSS_FUZZ_DIR)
         break
       except sp.CalledProcessError as e:
-        # Remove \n for single log entry on cloud.
+        # Replace \n for single log entry on cloud.
         stdout = e.stdout.decode('utf-8').replace('\n', '\t')
         stderr = e.stderr.decode('utf-8').replace('\n', '\t')
-        if (any(error in stdout + stderr for error in self._RETRYABLE_ERRORS)
-            and attempt_id < CLOUD_EXP_MAX_ATTEMPT):
+
+        captured_error = next(
```

Comment:
A silly question: Why do we use `next` instead of `any`?

Reply:
Previously we just need to check if any error in the list presents, so using `any` was fine. 
Now we also want to check for one specific case to add jitter, hence changing to `next` to get which error we are catching.
If we want to capture more errors with different backoff time in the future, would probably just refactor this in a separate function to simplify the code.

Reply:
> Now we also want to check for one specific case to add jitter, hence changing to next to get which error we are catching.

Should we iterate through items instead of only getting the first one via `next`?

> If we want to capture more errors with different backoff time in the future, would probably just refactor this in a separate function to simplify the code.

Yes, that's better.

Reply:
> Should we iterate through items instead of only getting the first one via `next`?

I think getting the first one is sufficient, but probably better to check the one with the longest backoff first